src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLogger.cs

```diff
@@ -416,7 +416,9 @@ private static void ApplyFunctionResultActivityTags(IEnumerable<KeyValuePair<str
                     currentActivity.AddTag(LogConstants.LogLevelKey, logLevel.ToString());
                 }
 
-                if (scope.TryGetValue(ApplicationInsightsScopeKeys.HttpRequest, out var request) && request is HttpRequest httpRequest)
+                if (!_loggerOptions.HttpAutoCollectionOptions.EnableHttpTriggerExtendedInfoCollection && 
+                    scope.TryGetValue(ApplicationInsightsScopeKeys.HttpRequest, out var request) &&
+                    request is HttpRequest httpRequest)
                 {
                     currentActivity.AddTag(LoggingConstants.ClientIpKey, GetIpAddress(httpRequest));
```

Comment:
So is `GetIpAddress()` just not working anymore?

Reply:
it's only working if `EnableHttpTriggerExtendedInfoCollection` is disabled (explicitly) - in this case AppInsights won't be able to collect any extra info about requests.  So, I'd say it's there for back-compat. 

What do you think, should we keep it? 
`

Reply:
Yeah, I think we should just keep it. I thought this originally worked when we did the Http tracking (without this PR) -- I was curious if it stopped somehow.